instrumentation-tests/src/androidTest/java/com/mapbox/navigation/instrumentation_tests/core/RouteAlternativesTest.kt

```diff
@@ -198,106 +199,111 @@ class RouteAlternativesTest : BaseTest<EmptyTestActivity>(EmptyTestActivity::cla
      */
     @Test
     fun additional_alternative_is_not_force_to_invoke_alternatives_observer() {
-        // Prepare with alternative routes.
-        setupMockRequestHandlers(coordinates)
-        val routes = requestDirectionsRouteSync(coordinates)
-
-        // Play primary route
-        runOnMainSync {
-            mapboxNavigation.historyRecorder.startRecording()
-            mockLocationReplayerRule.playRoute(routes.first())
-            mapboxNavigation.startTripSession()
-        }
+        IdlingPolicies.setIdlingResourceTimeout(1, TimeUnit.MINUTES)
```

Comment:
@RingerJK this is a complex test that also relies on an unmocked response data for alternative routes and it looks like the latest version of Nav Native returns a different set of alternatives so we actually need more time to reach the fork point so that the second alternative calls triggers. I'm increasing the timeout so that we have enough time to reach the fork point and recalculate alternatives before the test times out but I think we should refactor this whole test and abstract the response data.

Reply:
Yeah, let me increase the timeout in https://github.com/mapbox/mapbox-navigation-android/pull/6656 although we were lucky and last run passed 😅

Reply:
The unmocked response brought a lot of pain in https://github.com/mapbox/mapbox-navigation-android/pull/6628.